src/Raven.Studio/typescript/components/pages/database/tasks/list/panels/ReplicationHubConnectedSinkPanel.tsx

```diff
@@ -9,7 +9,7 @@ interface ReplicationHubPanelProps {
     data: OngoingTaskReplicationHubInfo;
 }
 
-export function ReplicationHubPanel(props: ReplicationHubPanelProps) {
+export function ReplicationHubConnectedSinkPanel(props: ReplicationHubPanelProps) {
```

Comment:
rename props type to match. There is convention there. If functional component is named x then props usually has type: xProps

Reply:
ok - good to know - done